FitNesseRoot/DbFit/AcceptanceTests/JavaTests/OracleTests/LibraryTests/StoredProcWithException/content.txt

```diff
@@ -1,12 +1,12 @@
-!3 Use ''Execute Procedure With Exception'' to check if a stored proc will throw an exception
+!3 Use ''Execute Procedure Expect Exception'' to check if a stored proc will throw an exception
 
 !|Execute|!-
 create or replace procedure TestProc1(name varchar2, strlength out number) as
 begin
-        if (name='xx') then
-          raise_application_error(-20001,'test exception');
-        end if;
-	strlength:=length(name);
+    if (name='xx') then
```

Comment:
If we're touching this, might be good to refine a few other things: add space around `=`, `:=`, and after `,`.